libnmstate/ifaces/ifaces.py

```diff
@@ -325,6 +326,17 @@ def _validate_unknown_parent(self):
                     f"{iface.parent}"
                 )
 
+    def _remove_unknown_interfaces(self):
+        """
+        Remove unknown type interfaces that come from the current state.
```

Comment:
`Remove unknown type interfaces`

Reply:
I've renamed the function name to `_remove_unknown_type_interfaces`. IMO, the description is fine as we are only removing the ones coming from the current state, not the desired state.

Reply:
Done, thanks!